credsweeper/common/morpheme_checklist.txt

```diff
@@ -584,7 +582,7 @@ fire
 firm
 first
 fix
-flash
+flas
 flat
```

Comment:
Could you explain why the keyword list modified?
Is there any data before and after?

Reply:
'ash' was added and it conflicts with 'flash', 'hash' ...

Reply:
Okay, then why the `ash` is added?
I'm not talking about the `flash` case only, but all.

Reply:
I don't remember exactly.
- it helps filter out 'hash_xxxxx...' creds - exist 'has' and 'ash'
- 'ash' is an independent word.